Fix precompile cache action in backend

JBlocks opened the pull request:
The precompile backend action (system toolbar) is now compatible with the request and response interface

nsd0jboesche committed:
[BUGFIX] Fix null result for count in warm up command

nsd0jboesche committed:
[FEATURE] Add request and response interface to fluid cache regenerate backend action

NamelessCoder commented:
This changes the TYPO3 compatibility of the extension and shouldn't enable strict types or declare strict returns. AFAIK it is possible to have compatible signatures for both v9 and v8 LTS versions.

JBlocks committed:
Update FluidPrecompileController.php

JBlocks committed:
Update FluidCacheRegenerateCacheAction.php

JBlocks committed:
Update FluidPrecompileController.php

JBlocks commented:
> This changes the TYPO3 compatibility of the extension and shouldn't enable strict types or declare strict returns. AFAIK it is possible to have compatible signatures for both v9 and v8 LTS versions.

Okay. Removed the `declare(strict_types=1);`, strict return type and added parameter signature `null` to get compatible with version 8. Version 8 has no middleware and no request and response interface as method signature. So we have $request = null and $response = null in version 8. I'm totally fine with this implementation.
